ZaeXT/test-scripts/test_cascade_delete.py

```
@@ -0,0 +1,130 @@
+        try:
+            print(f"      响应状态码: {response.status_code}, 内容: {response.json()}")
+        except json.JSONDecodeError:
+            print(f"      响应状态码: {response.status_code}, 内容: {response.text}")
+    exit(1)
+
+def get_auth_headers(token):
+    return {"Authorization": f"Bearer {token}", "Content-Type": "application/json"}
+
+def register_and_login(username, password):
+    """Registers and logs in a user, returning their token."""
+    requests.post(f"{BASE_URL}/register", json={"username": username, "password": password})
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)